content/LCDC.md

```diff
@@ -49,11 +49,13 @@ one.
 
 #### LCDC.5 - Window enable
 
-This bit controls whether the window shall be displayed or not. (TODO:
-what happens when toggling this mid-scanline ?) This bit is overridden
-on DMG by [bit 0](#lcdc-0-bg-and-window-enable-priority)
+This bit controls whether the window shall be displayed or not.
+This bit is overridden on DMG by [bit 0](#lcdc-0-bg-and-window-enable-priority)
 if that bit is reset.
 
+Behavior of this register when changing it mid-frame is a bit complicated;
```

Comment:
We should avoid giving opinions on how "complicated" something is. It's not necessary and it doesn't help the explanation. Mentioning that additional description/information is available in the linked paragraph is more than enough, IMO.

Reply:
"more complex", perhaps?